src/base/object_representation.h

```diff
@@ -695,19 +685,11 @@
    & OR_MVCC_REPID_MASK)
 
 /* in MVCC, chn follow by rep_id and/or ins_id depending by flags */
-#define OR_GET_MVCC_CHN(ptr, mvcc_flags) \
-  (((mvcc_flags) & OR_MVCC_FLAG_VALID_DELID) \
-   ? NULL_CHN \
-   : (((mvcc_flags) & OR_MVCC_FLAG_VALID_INSID) \
-      ? (OR_GET_INT (((char *) (ptr)) + OR_REP_OFFSET + OR_MVCC_REP_SIZE + OR_MVCCID_SIZE)) \
-      : ((OR_GET_INT (((char *) (ptr)) + OR_REP_OFFSET + OR_MVCC_REP_SIZE)))))
-
 #define OR_GET_MVCC_CHN_OFFSET(mvcc_flags) \
-  (((mvcc_flags) & OR_MVCC_FLAG_VALID_DELID) \
-   ? -1 \
-   : (((mvcc_flags) & OR_MVCC_FLAG_VALID_INSID) \
-      ? (OR_REP_OFFSET + OR_MVCC_REP_SIZE + OR_MVCCID_SIZE) \
-      : (OR_REP_OFFSET + OR_MVCC_REP_SIZE)))
+  (OR_REP_OFFSET + OR_MVCC_REP_SIZE + ((mvcc_flags) & OR_MVCC_FLAG_VALID_INSID?OR_MVCCID_SIZE:0) + ((mvcc_flags) & OR_MVCC_FLAG_VALID_DELID?OR_MVCCID_SIZE:0))
```

Comment:
The comment must be updated :smile: 
Please also beautify a little this macro. The conditions should be enclosed in parantheses and it would be more clear with spaces around '?' and ':' .
e.g. `((mvcc_flags) & OR_MVCC_FLAG_VALID_INSID?OR_MVCCID_SIZE:0)` -> `(((mvcc_flags) & OR_MVCC_FLAG_VALID_INSID) ? OR_MVCCID_SIZE : 0)`